_posts/2023-06-01-trino-fest-hype-speaker-lineup.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Refreshing at the lakehouse summer camp"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+show_pagenav: false
+---
+
+Trino Fest is just around the corner! We're only two weeks away from the day
+this post goes live, and we're excited to share that we've got an incredible
+speaker lineup with a wide variety of talks about all things Trino. If you're
+out of the loop, [we announced Trino Fest](/2023-04-05-announcing-trino-fest-2023.html)
+back in April as a two-day, free, virtual event. If you want to attend, see
+talks live, and engage with our speakers in Q&As at the end of each session,
+[go register now!](https://www.starburst.io/info/trinofest/) You do need to
+register to view the event, so don't delay.
+
+With that said, we're also excited to bring you a preview of our exciting
+speaker lineup. Read on if you'd like to hear more.
+
+<!--more-->
+
+## New connectors
+
+We've got two talks from Bloomberg and Redis about ongoing efforts to extend
+Trino's functionality to serve even more data sources. Erik Anderson and Yu Teng
+from Bloomberg will be talking about ForePaaS, a data platform that relies on
+Trino, as well as efforts to add a Snowflake connector to Trino to help power
+both ForePaaS and Trino up. Allen Terleto and Julien Ruaux at Redis will be
+talking about a new, custom, and improved Redis connector for Trino, showing how
+you can leverage the speed of both Redis and Trino to run queries faster than
+ever while seamlessly integrating with data visualization frameworks.
+
+## The Python ecosystem
+
+We've got talks from [Fugue](https://github.com/fugue-project/fugue) and
+[Ibis](https://ibis-project.org/), two different tools that integrate Python
+with SQL, and then run that SQL on underlying data sources. Both have recently
+added Trino support, and they're excited to share their use cases and introduce
+the Trino community to the new, powerful ways you can leverage it. Trino has
+always been a SQL query engine, but with Fugue and Ibis, writing Python code to
+run Trino is suddenly a reality, and analysts and data scientists may not even
+need to know much SQL to get the insights they're looking for.
+
+## Data lakes
+
+Ryan Blue, the co-founder of Iceberg working at Tabular, will be exploring how
+to best write CDC (change data capture) change streams into Iceberg tables. A
+talk from Kevin Liu at Stripe will explore how a data engineer can monitor
+queries being run on Iceberg to catch performance outliers and understand usage
+rates. A talk from Alluxio highlights caching optimizations with Trino and data
+lakes. OneHouse is giving a talk about using Trino with Hudi, exploring how to
```

Comment:
Should we link to all these companies??

Reply:
I don't think that adds a ton of value / utility. If people are curious, they can just google.